x/incentives/abci_test.go

```diff
@@ -30,6 +30,12 @@ func TestPerpetualGaugeNotExpireAfterDistribution(t *testing.T) {
 		Denom:         "lptoken",
 		Duration:      time.Second,
 	}
+
+	// mints coins so supply exists on chain
+	mintLPtokens := sdk.Coins{sdk.NewInt64Coin(distrTo.Denom, 200)}
+	err = simapp.FundAccount(app.BankKeeper, ctx, addr, mintLPtokens)
+	require.NoError(t, err)
+
```

Comment:
I think maybe the denom `"lptoken"` might be a bit misleading - I didn't pick apart what the entire test is doing and whether or not it is necessary for it to create a gauge that distributes to the denom `"lptoken"`, but since it does, we have to make sure the denom has supply on-chain or `CreateGauge` fails due to the new check I added. It doesn't affect the test's direct functionality - it just makes sure that it's compliant with the new constraint that we've added to `CreateGauge`.